core/crypto/vrf/vrf_types.hpp

```diff
@@ -17,6 +20,15 @@ namespace fc::crypto::vrf {
   using VRFResult = bls::Signature;
   using VRFHash = common::Hash256;
 
+  /**
+   * @brief vrf parameters structure
```

Comment:
Please comment the parameters in the context of VRF. Personally I don't fully get them.